pkg/network/node/networkpolicy.go

```diff
@@ -794,6 +798,35 @@ func (np *networkPolicyPlugin) parsePeerFlows(npns *npNamespace, npp *npPolicy,
 				// checking the source VNID...
 				npp.watchesNamespaces = true
 				peerFlows = append(peerFlows, np.selectNamespaces(peer.NamespaceSelector, dir)...)
+				if _, ok := os.LookupEnv(migrtionEnvVar); ok {
```

Comment:
do the `LookupEnv` in `NewNetworkPolicyPlugin` and stash the result in a field on `networkPolicyPlugin`